processor/package.json

```diff
@@ -1,17 +1,24 @@
 {
   "name": "@azure/event-processor-host",
-  "version": "1.0.6",
+  "version": "2.0.0-preview.1",
```

Comment:
Any reason for the version update?
We want to do this for event hubs as using the 1.0.0 version of amqp-common can break things for user.
Unless we increment the version of event hubs used in event-processor-host, I dont believe we need to update the version here

Reply:
You are right. There's no need to increase the major version. I will revert the change

Reply:
Reverted. We can increase the minor/patch number when we have a non-breaking future release.